container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3211,7 +3211,7 @@ variables:
     required: true
 - name: DEFAULT_STACK
   options:
-    default: *the-stack-name
+    default: "*the-stack-name"
```

Comment:
This is actually intentional, it gets pulled in from [here](https://github.com/SUSE/scf/blob/d4858f0d424fbd008f34d9a9ef3e04ad9256cc1f/container-host-files/etc/scf/config/opinions-opensuse42.yml) via [cat](https://github.com/SUSE/scf/blob/d4858f0d424fbd008f34d9a9ef3e04ad9256cc1f/bin/fissile#L33).  That's kind of janky, but we don't currently have a better solution.

Reply:
dropped.